src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -165,14 +165,12 @@ export const capitalizeEachWord = name => {
   }
 
   if (typeof name !== 'string') {
-    // Sentry.captureMessage(
-    //   `form_526_v1 / form_526_v2: capitalizeEachWord requires 'name' argument of type 'string' but got ${typeof name}`,
-    // );
+    Sentry.captureMessage(
```

Comment:
:man_shrugging: My two cents is to nix this whole chunk and handle the returned `null` appropriately from where this function is called without doing a type check each time.
```js
const capitalizedDisabilityName = capitalizeEachWord(disabilityName) || UNKNOWN_CONDITION;
```

If I recall, `Sentry.captureMessage()` doesn't return a stacktrace, so this wouldn't be useful for hunting down _where_ we're passing `capitalizeEachWord` a non-string. (I could be wrong there, mind you, but I _think_ all we get is the breadcrumbs from the browser—what buttons they poked, etc.)